hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -2125,22 +2119,12 @@
         It has the following attributes and sub-elements:
         * enabled:
             Set to true to enable Hazelcast's security features.
-        * <member-credentials-factory>:
-            Specifies the name and properties of your class that you developed by implementing Hazelcast's Credentials
-            interface.
-        * <member-login-modules>:
-            Specifies the name and properties of your class that you developed by implementing Hazelcast's LoginModule
-            interface. Each login module class can be given by using the <login-module> element. This element has the
-            attribute "usage" that specifies the way you want to use the LoginModule's  behavior during the
-            authentication process for Hazelcast members. Available values for this attribute are REQUIRED, REQUISITE,
-            SUFFICIENT and OPTIONAL.
-        * <client-login-modules>:
-        It has the same purpose as <member-login-modules> explained above. However, instead of Hazelcast members,
-        it is used for Hazelcast native clients.
-        * <client-permission-policy>:
-            Specifies the name and properties of your class that you developed by implementing Hazelcast's
-            IPermissionPolicy interface, which is the default permission policy (com.hazelcast.security.IPermissionPolicy).
-            This policy defines the client authorization specifications.
+        * <realms>
+            Defines named security realms.
+        * <memmber-authentication>
```

Comment:
me**mm**ber - have you, by any chance, switched to a Macbook Pro with its great keyboard?

Reply:
Nope, fortunately not :) It's just my own ignorance for typos ;)
:+1: